src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -19,6 +19,7 @@
     <Reference Include="Microsoft.AspNetCore.StaticFiles" />
     <Reference Include="Microsoft.AspNetCore.SpaServices.Extensions" />
     <Reference Include="Microsoft.Extensions.FileProviders.Embedded" />
+    <Reference Include="Microsoft.Extensions.TypeNameHelper.Sources" PrivateAssets="All" />
```

Comment:
I'm curious about what this is for.

Reply:
It was for pretty printing the component name. But now that we no longer include it, this reference is gone.